music21/converter/subConverters.py

```diff
@@ -1015,6 +1015,65 @@ def show(self, obj, fmt, app=None, subformats=None, **keywords):  # pragma: no c
         self.launch(returnedFilePath, fmt=fmt, app=app)
 
 
+# ------------------------------------------------------------------------------
+class ConverterFlatio(SubConverter):
+    '''
+    Converter for Flatio. Sends XML to http://flat.io and then opens it in a browser tab.
+
+    You must have a flat.io authentication token, and set it like
+    >>> us = environment.UserSettings()
+    >>> us['flatioAuthToken'] = '[MY FLAT.IO AUTH TOKEN]'
+
+    See https://flat.io/developers/docs/api/authentication.html#personal-access-tokens
+    '''
+    registerFormats = ('flat',)
+    registerOutputExtensions = ('flat',)
+
+    def parseData(self, strData, number=None):
+        '''
+        Not implemented yet.
+        '''
+        # Pull MusicXML from a flat.io url?
+
+    def write(self, obj, fmt, fp=None, subformats=None,
+              compress=False, **keywords):  # pragma: no cover
+        '''
+        Uploads MusicXML to flat.io via their API.
+        '''
+        import requests
```

Comment:
requests isn't in the requirements.txt.  Is it possible to do this with the python standard library?  If not, it needs to be added to requirements.txt

Reply:
Oops, was already mentioned above.  I think that given that the Python std library docs already suggest using requests instead, and it's only not in the std library because it gets security fixes faster than other libraries, it's probably okay.  So we just need the format name changed and a mechanism for helping users get their API codes, etc.